.common.sh

```diff
@@ -93,9 +93,11 @@ exec_ssoadm() {
 check_am() {
   local instance_id=${1:-1}
   local expect_alive=${2:-1}
-  local status=$(exec_am $instance_id curl -s http://localhost:8080/auth/isAlive.jsp)
```

Comment:
Exit code of this line is always 0, because the exit code comes from "local" bulitin. This can cause false positives when Wren:AM instance is not running yet.